.github/workflows/stale.yaml

```diff
@@ -0,0 +1,42 @@
+name: 'Close stale issues and PRs'
+
+on:
+  push:
+    paths:
+      - .github/workflows/stale.yml
+    branches-ignore:
+      - dependabot/**
+  schedule:
+    # Once every day at midnight UTC. Source: https://crontab.guru/every-day
+    - cron: '0 0 * * *'
+
+permissions:
+  issues: write
+  pull-requests: write
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v9
+        with:
+          # Setting this variable to 'true' will cause the
+          # action to run in a dry-run mode
+          debug-only: ${{ vars.DEBUG_ONLY == 'true' }}
```

Comment:
Where will this variable value come from? 🤔

Reply:
We'll create a variable when we want it to run in debug only/dry-run mode - https://github.com/warm-metal/csi-driver-image/pull/114#issuecomment-1890906311.

Otherwise the action will run fully against the issues/PRs

Reply:
Done, added the variable.

Reply:
You can delete the variable after at least one dry run.